scripts/process-scripts.py

```diff
@@ -215,7 +215,8 @@ def make_tables():
                     # get content on right side of "="
                     val = obj.value
                     if val.lineno == val.end_lineno:
```

Comment:
This should actually catch statements that are actually on one line
```suggestion
                    if obj.lineno == obj.end_lineno:
```

Reply:
I'm new to github and still learning how this PR system works.

I ran locally all the tests and they passed after using black on the example scripts. I force pushed that change and lets see if the PR updates and passes with your suggestion.